main/src/ca/uwaterloo/flix/language/phase/Monomorph.scala

```diff
@@ -226,92 +226,13 @@ object Monomorph extends Phase[TypedAst.Root, TypedAst.Root] {
           val es = exps.map(visitExp(_, env0))
           Expression.Apply(e, es, subst0(tpe), eff, loc)
 
-        case Expression.UnaryDeprecated(op, exp, tpe, eff, loc) =>
-          val e1 = visitExp(exp, env0)
-          Expression.UnaryDeprecated(op, e1, subst0(tpe), eff, loc)
-
         case Expression.Unary(sop, exp, tpe, eff, loc) =>
           val e1 = visitExp(exp, env0)
           Expression.Unary(sop, e1, subst0(tpe), eff, loc)
 
-        /*
-         * Equality / Inequality Check.
-         */
-        case Expression.BinaryDeprecated(op@(BinaryOperator.Equal | BinaryOperator.NotEqual), exp1, exp2, tpe, eff, loc) =>
-          // Perform specialization on the left and right sub-expressions.
-          val e1 = visitExp(exp1, env0)
-          val e2 = visitExp(exp2, env0)
-
-          // The type of the values of exp1 and exp2. NB: The typer guarantees that exp1 and exp2 have the same type.
-          val valueType = subst0(exp1.tpe)
-
-          // The expected type of an equality function: a -> a -> bool.
-          val eqType = Type.mkPureUncurriedArrow(List(valueType, valueType), Type.Bool)
-
-          // Look for an equality function with the expected type.
-          // Returns `Some(sym)` if there is exactly one such function.
-          lookupEq(eqType) match {
```

Comment:
Does this mean that lookupEq is now unused and can be removed? or not yet?

Reply:
Yes it does, I missed that. I'll remove it.